src/admin/editModal.component.js

```diff
@@ -34,47 +51,34 @@ function EditClassModal(props) {
                     </Button>,
                 ]}
             >
-                <Row>
-                    <Col span={6}>Title:</Col>
-                    <Col span={12}><input type='text' placeholder='Title' name='title' value={title} onChange={e => setTitle(e.target.value)}></input></Col>
-                </Row>
-                <Row>
-                    <Col span={6}>Year of Class:</Col>
-                    <Col span={16}><input type='text' placeholder='Year of Class' name='yearOfClass' value={yearOfClass} onChange={e => setYearOfClass(e.target.value)}></input></Col>
-                </Row>
-                <Row>
-                    <Col span={6}>Module Number:</Col>
-                    <Col span={12}><input type='text' placeholder='Module Number' name='moduleNumber' value={moduleNumber} onChange={e => setModuleNumber(e.target.value)}></input></Col>
-                </Row>
-                <Row>
-                    <Col span={6}>Difficulty:</Col>
-                    <Col span={12}><input type='text' placeholder='Difficulty' name='difficulty' value={difficulty} onChange={e => setDifficuty(e.target.value)}></input></Col >
-                </Row>
-                <Row>
-                    <Col span={6}>Language:</Col>
-                    <Col span={12}><input type='text' placeholder='Languages' name='tags' value={tags} onChange={e => setTags(e.target.value)}></input></Col>
-                </Row>
+                <Form {...formItemLayout}>
+                    <Form.Item label="Title"><Input type='text' name='title' value={title} onChange={e => setTitle(e.target.value)}></Input></Form.Item>
+                    <Form.Item label="Year of Class"><Input type='text' name='yearOfClass' value={yearOfClass} onChange={e => setYearOfClass(e.target.value)}></Input></Form.Item>
+                    <Form.Item label="Module Number"><Input type='text' name='moduleNumber' value={moduleNumber} onChange={e => setModuleNumber(e.target.value)}></Input></Form.Item>
+                    <Form.Item label="Difficulty"><Input type='text' name='difficulty' value={difficulty} onChange={e => setDifficuty(e.target.value)}></Input></Form.Item>
+                    <Form.Item label="Languages">
+                        <Select
```

Comment:
you will want to set the value prop of this to be what is currently in state - notice how when you edit a class, the languages are not set to what they were